OWNERS

```diff
@@ -0,0 +1,7 @@
+reviewers:
+  - ghatwala
+  - valen-mascarenhas14
+approvers:
+  - ghatwala
+  - valen-mascarenhas14
+  
```

Comment:
I still see this 
<img width="113" alt="image" src="https://github.com/user-attachments/assets/12328f93-3312-4912-8f97-780ce4c64ed9">
symbol!

Reply:
fixed it @mkumatag